x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/saved_object/engine_descriptor.ts

```diff
@@ -78,7 +78,18 @@ export class EngineDescriptorClient {
     return attributes;
   }
 
-  async update(entityType: EntityType, status: EngineStatus) {
+  async update(entityType: EntityType, engine: Partial<EngineDescriptor>) {
+    const id = this.getSavedObjectId(entityType);
+    const { attributes } = await this.deps.soClient.update<EngineDescriptor>(
+      entityEngineDescriptorTypeName,
+      id,
+      engine,
+      { refresh: 'wait_for' }
+    );
+    return attributes;
+  }
+
+  async setStatus(entityType: EntityType, status: EngineStatus) {
```

Comment:
`setStatus` could be called `UpdateStatus` or better... Delete it and replace it with an `update` call.
If that is too much for FF, it is also possible to call `update` from inside the `updateStatus` method.

Reply:
I can call update from inside the method.
I think it's such a common action it deserves its own method